services/processor/native/service.go

```diff
@@ -70,5 +67,11 @@ func (s *service) DeployNativeService(input *services.DeployNativeServiceInput)
 }
 
 func (s *service) RegisterContractSdkCallHandler(handler handlers.ContractSdkCallHandler) {
```

Comment:
I don't get this method. Why isn't this being done on construction?

Reply:
This method will be called on the construction of `VirtualMachine`, this part wasn't implemented yet